Node.md

```diff
@@ -1,10 +1,10 @@
-# Node & Viam's TypeScript SDK
+# Node.js & Viam's TypeScript SDK
 
-This document contains detailed instructions on including Viam in your Node project. For a runnable example, see the [examples directory](/examples/node/).
+This document contains detailed instructions on including Viam in your Node.js project. For a runnable example, see the [examples directory](/examples/node/).
 
 ## Requirements
 
-This document assumes you already have Node installed. If not, follow the [instructions](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs) provided by Node.
+This document assumes you already have Node.js >= version 20 installed. If not, follow the [instructions](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs) provided by Node.js.
```

Comment:
The `--env-file` flag was introduced in v20, so calling this out just in case some folks are running v18 which is still under long term support.